docker/Dockerfile

```diff
@@ -38,7 +38,8 @@ ARG DEBIAN_FRONTEND
 ARG GOPATH
 
 # Set HOME to a writable directory in case something wants to cache things
-ENV HOME=/tmp
+ENV HOME=/scratch
+ENV TMP=/scratch
```

Comment:
/scratch is very specific to how the Collabora gitlab runners are setup; So they shouldn't be set as defaults. iotw please drop this chunk :)

Reply:
done; uml complains if $TMP isn't set (or the default) so I've set that to `/tmp`